test/Unsafe/safe.swift

```diff
@@ -218,6 +220,10 @@ func yetMoreUnsafeFunc(unsafe: () -> Void) {
   // expected-warning@-1{{no unsafe operations occur within 'unsafe' expression}}
 }
 
+func yetMoreMoreUnsafeFunc(unsafe: Int?) {
+  _ = unsafe! + 1
```

Comment:
There should be a separate test for `unsafe + 1` I think, otherwise this is only testing the postfix logic

Reply:
Thank you!
What if we do it like this? https://github.com/swiftlang/swift/pull/81429/commits/1fe8ac8c56a12b538ab53659d05e84abf28a0b2e

Reply:
Looks good!